<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Calibri</family>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Reacting!!!</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/icon/IMG_2775(20200519-194904).JPG</normalon>
   </iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">border-color: rgb(170, 255, 0);
background-color: rgb(170, 255, 255);</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Reacting_button">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>280</y>
      <width>371</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Brush Script MT</family>
      <pointsize>48</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
background-color: rgb(255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Reacting!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>28</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Reacting test</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Start_button">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>190</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(255, 255, 0);
background-color: rgb(255, 255, 0);</string>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Scoreboard_button">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>190</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Scoreboard</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Quit_button">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>190</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 0);</string>
    </property>
    <property name="text">
     <string>QUIT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>511</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Press START button (or press key S) to start the test
When the surface turn green, it means that the test has started.
When the surface turn red, press the 'Reacting' button (or press key SPACE) as soon as possible!
</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>731</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 16pt &quot;Bradley Hand ITC&quot;;

</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Window_icon.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>Quit_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>490</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
